setup.py

```diff
@@ -48,22 +48,30 @@
         'Topic :: Software Development :: Libraries :: Python Modules',
     ],
     install_requires = [
-        'Django==1.8.13',
-        'django-autocomplete-light==2.3.3',
-        'django-extensions>=1.5.5',
-        'django-nose>=1.4',
+        'Django>=1.8.2',
```

Comment:
As you said, we have some technical debt and there was architectural mistake made, which we have to deal with. I totally agree with you, that django-powerdns-dnssec (apart from the name of this project, which is terrible), per se, should not specify django version, but only if it'll be simple "overlay" for powerdns.

IMO the good way of designing this app is as follows:
* powerdns app is simple overaly to powerdns database - it basically has only models (and probably admin). This app does not require any specific django version (it might support Django 1.7+ as you said).
* dnsaas app, on the other hand, provides "aaS" models (like Request etc; if there is need to extend orignal PowerDNS it should be done by model inheritance here) and API (? - not sure if simple API should not be part of powerdns). This app is dependent on powerdns app and it *might* require particular django version.
* ui could be other app, dependent on dnsaas and using it's API.

To make it reasonable, we should split it into two projects (repositories) - django-powerdns-dnssec (which is basically reflection of PowerDNS into Django models, and DNSaaS, which is self-service for managing DNS records based on django-powerdns-dnssec.

But... coming down to earth.. I'm afraid that some code of this project is not compatible with Django 1.9 or 1.10 - I'd be happy to bump it, but IMO it should be well tested first.